src/ck-perf/trace-common.C

```diff
@@ -19,6 +19,14 @@
 #include "allEvents.h"          //projector
 #include "register.h" // for _entryTable
 
+// To get username
+#if defined(_WIN32) || defined(_WIN64)
```

Comment:
Not terribly important, but `#ifdef _WIN32` would be sufficient here. It is present for both 32-bit and 64-bit.